pytab/api.py

```diff
@@ -70,13 +70,13 @@ def getTestData(platformID: str) -> None:
     "ffmpeg": {
         "ffmpeg_source_url": "https://repo.jellyfin.org/releases/server/windows/versions/jellyfin-ffmpeg/5.1.4-3/jellyfin-ffmpeg_5.1.4-3-portable_win64.zip",
         "ffmpeg_version": "5.1.6-3",
-        "ffmpeg_hashs": None,
```

Comment:
I know, the whitepaper actually writes "hashs", but is wrong the hash plural is hashes

Reply:
Since this is not a local used variable but will actualy be part of the Server Return, this cannot be changed here!
If the Whitepaper changes, we can do that!
But thats not on me!

We cannot change the client to expect / deliver variables with correct spelling, if the front-end doesnt support it